app/importer.js

```diff
@@ -233,30 +232,6 @@ const shouldSkipCookie = (cookie) => {
 }
 module.exports.shouldSkipCookie = shouldSkipCookie
 
-importer.on('add-cookies', (e, cookies) => {
```

Comment:
as mentioned in https://github.com/brave/muon/issues/320#issuecomment-340082014. we should also remove shouldSkipCookie and its unittest